modules/zopfli/1.0.3.bcr.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,64 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,25 @@
++cc_library(
++    # Named to produce "libzopfli.{a,so}"
++    # avoiding https://github.com/bazelbuild/bazel-central-registry/issues/716
++    name = "zopfli",
++    srcs = glob(["src/zopfli/*.c"], exclude = ["**/*_bin.c"]),
++    hdrs = glob(["src/zopfli/*.h"]),
++    linkopts = ["-lm"],
++    strip_include_prefix = "src/",
++    visibility = ["//visibility:public"],
++)
++
++cc_binary(
```

Comment:
The targets below don't seem to be visible. Would users need them?

Reply:
I _personally_ don't need them, but I can't speak for all use cases.

I _think_ they exist solely to service the `zopflipng` cli, but I admit I'm not totally certain of that.